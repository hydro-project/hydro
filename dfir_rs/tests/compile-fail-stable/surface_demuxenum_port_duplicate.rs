../compile-fail/surface_demuxenum_port_duplicate.rs