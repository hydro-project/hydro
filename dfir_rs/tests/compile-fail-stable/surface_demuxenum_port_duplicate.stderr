error: Output connection conflicts with below (unknown:0:0) (1/2)
  --> tests/compile-fail-stable/surface_demuxenum_port_duplicate.rs:20:18
   |
20 |         my_demux[Square] -> for_each(std::mem::drop);
   |                  ^^^^^^

error: Output connection conflicts with above (unknown:0:0) (2/2)
  --> tests/compile-fail-stable/surface_demuxenum_port_duplicate.rs:21:18
   |
21 |         my_demux[Square] -> for_each(std::mem::drop);
   |                  ^^^^^^
